### PR TITLE
Feedback Posts to Janis' repo Issues

### PR DESCRIPTION
**Pull request by ifsimicoded**
completes an additional step of #18.
- [ ] figure out how we want to log email addresses -- currently shows in issue, which isn't very private.
- [ ] determine what we want to do in production vs development mode.

### CODE REVIEW

File: scripts/serve-local.sh [hunk 1]
```diff
@@ -19,6 +19,7 @@ docker run \
     --volume "$PWD/static.config.js:/app/static.config.js" \
     --volume "$PWD/.babelrc:/app/.babelrc" \
     --volume "$PWD/yarn.lock:/app/yarn.lock" \
+    --env "FEEDBACK_API=https://coa-test-form-api.herokuapp.com/process/" \
```
Comment (mateoclarke) [inline scripts/serve-local.sh]: Should this be set as an environment variable if it is the same for all the environments?

File: src/js/page_sections/FormFeedback.js [hunk 2]
```diff
@@ -85,10 +88,20 @@ class FormFeedback extends Component {
...
-    // after successful form submit
-    this.setState({
-      stage: 0
+
+    postFeedback({
+      title: this.state.values['site-feedback-options'],
+      description: this.state.values['site-feedback-textarea'],
+      email: this.state.values['site-feedback-email']
+    })
+    .then(({data: data}) => {
```
Comment (mateoclarke) [inline src/js/page_sections/FormFeedback.js]: could this line just be `.then(({ data }) => {`

File: src/js/page_sections/FormFeedback.js [hunk 3]
```diff
@@ -269,7 +282,7 @@ class FormFeedback extends Component {
           <div className="coa-overlay">
             <div className="coa-overlay__content">
               <h4>Thank you for sharing your feedback!</h4>
-              <p>You can see your feedback in our <a href="#">austin.gov feedback tracker</a></p>
+              <p>You can see your feedback in our <a href={this.state.successUrl} target="_blank" rel="noopener noreferrer" aria-label="Opens in new window">austin.gov feedback tracker</a></p>
```
Comment (mateoclarke) [inline src/js/page_sections/FormFeedback.js]: 👍

**Comment by mateoclarke** [comment]
Full story form abandonment looks really cool.

**Comment by ifsimicoded** [comment]
Hey y'all, sorry I forgot to create a new branch, so I updated this PR with event tracking. If y'all could review.
https://github.com/cityofaustin/janis/pull/151/commits/460cba7201437e903fe90b62912d0768f084d619

File: src/App.js [hunk 4]
```diff
@@ -1,21 +1,21 @@
 import React, { Component } from 'react'
 import { Router, Route } from 'react-static'
 import ReactGA from 'react-ga'
-import GoogleAnalyticsPageView from 'js/helpers/GoogleAnalyticsPageView'
+import { logPageView } from 'js/helpers/googleAnalytics'
```
Comment (mateoclarke) [inline src/App.js]: I like this.

File: scripts/serve-local.sh (hunk 1 above)
Comment (ifsimicoded) [reply scripts/serve-local.sh]: I'm thinking we'll have different endpoints in different environments (with prod having more rigorous validation). We probably don't want issues being created in dev/staging as is currently the case, and I haven't thought through what the long term plan is, but this leaves it open. Also cool with changing -- lemme know what you're thinking.

File: src/js/page_sections/FormFeedback.js (hunk 2 above)
Comment (ifsimicoded) [reply src/js/page_sections/FormFeedback.js]: yep!

**Comment by ifsimicoded** [comment]
@mateoclarke if you have implementation changes, I'm down to talk/pair.

**Pull request merged**